pkg/model/profile_manager.go

```diff
@@ -0,0 +1,52 @@
+package model
+
+import (
+	"os"
+	"strings"
+
+	"github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
+)
+
+const (
+	RHSSOProfile                 = "RHSSO"
+	ProfileEnvironmentalVariable = "PROFILE"
+)
+
+var Profiles = NewProfileManager()
+
+type ProfileManager struct {
+	Profiles []string
+}
+
+func NewProfileManager() ProfileManager {
+	return ProfileManager{
+		Profiles: getProfiles(),
+	}
+}
+
+func (p *ProfileManager) IsRHSSO(cr *v1alpha1.Keycloak) bool {
+	for _, profile := range p.Profiles {
+		if profile == RHSSOProfile {
+			return true
+		}
+	}
+	if cr != nil && cr.Spec.Profile == RHSSOProfile {
+		return true
+	}
+	return false
+}
+
+func (p *ProfileManager) GetKeycloakOrRHSSOImage(cr *v1alpha1.Keycloak) string {
+	if p.IsRHSSO(cr) {
+		return Images.Images[RHSSOImage]
+	}
+	return Images.Images[KeycloakImage]
+}
+
+func getProfiles() []string {
+	env := os.Getenv(ProfileEnvironmentalVariable)
+	if env == "" {
+		return []string{}
+	}
+	return strings.Split(env, ",")
```

Comment:
What is the reason for having the possibility to set more profiles?

Reply:
At the moment, we don't need additional profiles. However, I'm not exactly sure what the productization process will bring us, so I would prefer to have a ready-to-go implementation at hand to add more if we need them. However, if you don't like it - I can remove it. I just have a gut feeling they might be useful. Nothing more.